packages/odyssey-react/.postcssrc.js

```diff
@@ -32,6 +32,10 @@ module.exports = (ctx) => {
         grid: "autoplace",
         env: "production",
       },
+      modules: {
```

Comment:
Updates scoped names to have the following format: ".ods-label-d3fa4s". This should help with tracking down relevant styles while troubleshooting UI. `name` and `folder` were left off here to keep classes shorter.

@jeffbelser-okta If we don't want to merge this change until we have a friendlier solve, lmk.

Reply:
Turns out this generates an `undefined.json` file in the react package directory. Removing for now.

Reply:
@edburyenegren-okta we won't want to merge this because it will affect our production build too. We want class names there to be short and sweet. We should be able to configure this postcssrc function to respond when in the context of a storybook build however and add this exact string so we have what we want on local dev